config.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Can we do a better job of naming this?  If it's for algolia, let's call it algolia-config.json